ext/em.cpp

```diff
@@ -877,7 +883,7 @@ _SelectDataSelect
 static VALUE _SelectDataSelect (void *v)
 {
 	SelectData_t *sd = (SelectData_t*)v;
-	sd->nSockets = rb_fd_select (sd->maxsocket+1, &(sd->fdreads), &(sd->fdwrites), &(sd->fderrors), &(sd->tv));
+	sd->nSockets = select (sd->maxsocket+1, &(sd->fdreads), &(sd->fdwrites), &(sd->fderrors), &(sd->tv));
```

Comment:
This has to use a raw `select` because it is invoked inside a "thread blocking region". This was a legacy api in ruby 1.9.x which allowed callbacks to be invoked outside ruby's GVL, and thus is not allowed to call back into any ruby apis.